src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -14,30 +16,36 @@ import useToggle from '@common/hooks/useToggle';
 import { mapFieldErrors } from '@common/validation/serverValidations';
 import Yup from '@common/validation/yup';
 import { Formik } from 'formik';
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 
 interface FormValues {
-  email: string;
+  emails: string;
 }
 
 const errorMappings = [
   mapFieldErrors<FormValues>({
-    target: 'email',
+    target: 'emails',
     messages: {
-      USER_ALREADY_EXISTS: 'User with this email already exists',
+      INVALID_EMAIL_ADDRESS:
+        'Please enter valid @education.gov.uk email addresses',
+      NO_INVITABLE_EMAILS:
+        'All of the email addresses are already invited or accepted users of the pre-release',
```

Comment:
Wording is a little confusing - perhaps 'All of the email addresses have already been invited or accepted'

I've excluded the additional part about pre-release because it's a bit redundant in the context of where the form is.

Reply:
Abbreviated this one to your suggested text. Thanks.